subsystem/cluster_resource_rosa_test.go

```diff
@@ -2563,7 +2563,7 @@ var _ = Describe("ocm_cluster_rosa_classic - upgrade", func() {
 					worker_role_arn = "",
 				}
 			}
-			version = "openshift-v4.10.1"
+			version = "4.10.1"
```

Comment:
Can you add a failure test that uses an incorrect version string?

Reply:
Please see new comment added https://github.com/terraform-redhat/terraform-provider-ocm/pull/124/commits/530b0881efb093ebfe052f6fd001164cba419894